ntoskrnl/io/iomgr/driver.c

```diff
@@ -1186,6 +1186,7 @@ IopInitializeSystemDrivers(VOID)
 NTSTATUS NTAPI
 IopUnloadDriver(PUNICODE_STRING DriverServiceName, BOOLEAN UnloadPnpDrivers)
 {
+    UNICODE_STRING Backslash = RTL_CONSTANT_STRING(L"\\");
```

Comment:
Is `/` not accepted?

Reply:
`DriverServiceName` is an NT path (`\Registry\Machine\System\ControlSet001\Services\abcde`), so forward slashes won't be accepted, no.
Though... that means this really should be `OBJ_NAME_PATH_SEPARATOR`.

Reply:
@ThFabba : Use OBJ_PATH_SEPARATOR constant?
@learn-more : In "core" NT it's only the backslash. The handling of slash character is only in win32 (and in ntdll) where it gets converted back to backslash in a "consistent" manner.